tests/language/abstract_beats_arguments2_test.dart

```diff
@@ -1,14 +0,0 @@
-// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
```

Comment:
Yes. In 2.0, a program that tries to construct an abstract class is statically erroneous.